azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -688,17 +688,22 @@ private void ajaxFetchJobInfo(final Project project, final HashMap<String, Objec
       return;
     }
 
-    final Props prop;
+    Props prop;
     try {
-      prop = this.projectManager.getProperties(project, node.getJobSource());
+      prop = this.projectManager.getProperties(project, flow, jobName, node.getJobSource());
     } catch (final ProjectManagerException e) {
       ret.put("error", "Failed to retrieve job properties!");
       return;
     }
 
+    if (prop == null) {
+      prop = new Props();
+    }
+
     Props overrideProp;
     try {
-      overrideProp = this.projectManager.getJobOverrideProperty(project, jobName);
+      overrideProp = this.projectManager
```

Comment:
Is this prop still for Job? Can we override Flow Properties in the long run?

Reply:
Here this overrideProp is for job override to keep consistent with the current implementation. But you are right, we can support overriding flow properties in the same way as well.